packages/babel-helpers/src/helpers/defaults.ts

```diff
@@ -0,0 +1,21 @@
+/* @minVersion 7.0.0-beta.0 */
+export default function _defaults<T extends object, S extends object>(
+  obj: T,
+  defaults: S,
+): NonNullable<T & S> {
+  for (
+    var keys: string[] = Object.getOwnPropertyNames(defaults), i = 0;
+    i < keys.length;
+    i++
```

Comment:
If I hoist the `keys` to the scope of the `_defaults`, the generated size will be increased by 3 ~ 5 bytes.